src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -89,14 +89,7 @@ void Snippet::copy_snippet() {
         auto new_input = std::make_shared<ngraph::opset1::Parameter>(input.get_element_type(), input.get_partial_shape());
         subgraph_node_inputs.push_back(new_input);
     }
-    std::shared_ptr<ov::Model> new_body = nullptr;
-    // Ticket[79554]: TypeRelaxed ops aren't thread safe so we use mutex to avoid collision in throughput mode
-    if (original_snippet->has_type_relaxed_ops()) {
-        std::lock_guard<std::mutex> lock(*context->getSharedMutex());
```

Comment:
I think we can remove `getSharedMutex()` function I see that we use this method only here.